cmd/ocagent/main.go

```diff
@@ -121,14 +121,15 @@ func runZPages(port int) func() error {
 		log.Fatalf("Failed to bind to run zPages on %q: %v", addr, err)
 	}
 
+	srv := http.Server{Handler: zPagesMux}
 	go func() {
 		log.Printf("Running zPages at %q", addr)
-		if err := http.Serve(ln, zPagesMux); err != nil {
+		if err := srv.Serve(ln); err != nil && err.Error() != "http: Server closed" {
```

Comment:
nit: is this really the best practice? isn't there a const defined somewhere?

Reply:
Ah, thanks, that was what I found at first for golang in general but the package itself defines the proper constant. I will update.